distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc

```diff
@@ -0,0 +1,32 @@
+:title: Installing from ${command-console}
+:type: installing
+:status: published
+:summary: Installing from ${command-console}.
+:project: ${branding}
+:order: 08
+
+===== Completing Installation from the ${command-console}
+
+In order to install ${branding} from the ${command-console}, use the command `profile:install <profile-name>`.
+The `<profile-name>` should be the desired <<_setup_types, Setup Type>> in lowercase letters.
+
+[NOTE]
+====
+This only installs the desired Setup Type. There are other components that can be set up in the ${admin-console} Installer that cannot be setup on the ${command-console}.
```

Comment:
❓ @ricklarsen , what is the general rule for links?  I see there is a link for Setup Type above but not here. Do we make a link for every occurrence?

Reply:
Not for every occurrence. I'd say as a rule of thumb, it depends on how close the occurrences are to each other. If there is some distance between this one and the last one, it makes sense to link them.

Reply:
Yeah, these occurrences are really close together so I think I'll just link on the first reference.